src/transport_controller/internals/pipe.go

```diff
@@ -0,0 +1,80 @@
+package internals
+
+import (
+	"net"
+
+	"github.com/HyperloopUPV-H8/Backend-H8/transport_controller/models"
+)
+
+type Pipe struct {
+	conn   *net.TCPConn
+	laddr  *net.TCPAddr
+	raddr  *net.TCPAddr
+	config models.Config
```

Comment:
what is laddr and raddr for?

Reply:
so it knows the addresses (laddr: local address, raddr: remote addres (the board)) when connecting to the board